ADOPTERS.md

```diff
@@ -0,0 +1,9 @@
+# Copa Adopters
```

Comment:
would we want to create this in the website instead of a standalone doc?

Reply:
I'll add it to the website as well, but an ADOPTERS.md file is a standard practice too.